build-lab.sh

```diff
@@ -27,6 +27,10 @@ sudo apt-get install -y tree
 
 echo -e "${LINE_ASCII_CONSOLE}\n"
 
+echo "gem: --no-ri --no-rdoc" >> ${lab}/.gemrc
+
+export CHEF_LICENSE="accept-silent"
```

Comment:
i've no problem with this in the pr but we should revise the text to mention that this change is also occurring